### PR TITLE
8.0 oca machine manager

### PR DESCRIPTION
**Pull request by mathi123**

### CODE REVIEW

File: machine_manager/models/machinery.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                               help="This association is necessary if you want"
+                               " to make repair orders with the machine")
+    enrolldate = fields.Date('Enrollment date', required=True,
+                             default=lambda
+                             self: fields.Date.context_today(self))
+    ambit = fields.Selection([('local', 'Local'), ('national', 'National'),
+                              ('international', 'International')],
+                             'Ambit', default='local')
+    card = fields.Char('Card')
+    cardexp = fields.Date('Card Expiration')
```
Comment (mathi123) [inline machine_manager/models/machinery.py]: @pedrobaeza What are these 'card' fields for?

File: machine_manager/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+##############################################################################
```
Comment (mathi123) [inline machine_manager/__init__.py:4]: @pedrobaeza Is it necessary to have this license statement on every file?

File: machine_manager/views/product_view.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="product_normal_form_view_inh" model="ir.ui.view">
```
Comment (mathi123) [inline machine_manager/views/product_view.xml]: @pedrobaeza In general, can I change Ids when porting modules to OCA? Or do you have these modules in a production environment?

File: machine_manager/models/machinery.py (hunk 1 above)
Comment (pedrobaeza) [inline machine_manager/models/machinery.py]: @Daniel-CA, what is the purpose of these fields?

File: machine_manager/__init__.py (hunk 2 above)
Comment (pedrobaeza) [inline machine_manager/__init__.py:4]: Answer already

File: machine_manager/views/product_view.xml (hunk 3 above)
Comment (pedrobaeza) [inline machine_manager/views/product_view.xml]: Answer also

File: machine_manager/models/machinery.py (hunk 1 above)
Comment (Daniel-CA) [inline machine_manager/models/machinery.py]: 'Card' field is used if the machine has any official document and its expiration date, maybe is not something usual.I can remove it if you think that does not apply.

Comment (pedrobaeza) [inline machine_manager/models/machinery.py]: It should be called then `License card`

**Pull request closed without merging**